api/src/app/app.module.ts

```diff
@@ -45,7 +45,6 @@ import { MultiSigModule } from "../multi-sig/multi-sig.module.js";
     StatsModule,
     WalletSubscriptionModule,
     OlSwapModule,
-    NodeWatcherModule,
```

Comment:
why?

Reply:
I noticed that the NodeWatcherModule was imported twice in the AppModule. This seemed to be a redundancy as importing a module once is enough for it to be available throughout the application. To keep the code clean and efficient, I removed the duplicate import. Please let me know if there was a specific reason for the double import that I might have overlooked.